From 1b2fff319084c57ffcb2f4e91dc35c6713c8d2b7 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Konrad=20Zapa=C5=82owicz?=
 <konrad.zapalowicz@chainguard.dev>
Date: Fri, 5 Sep 2025 11:27:55 +0200
Subject: [PATCH 3/4] refactor(entitlements): make Chainguard the default
 vendor
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

The entitlements client is forcing Control Plane as a default vendor.
Chainguard images are not reporting and being served through their
infrastructure and so the default vendor isn't accurate.

This commit is overriding the default vendor with Chainguart to make it better
reflect reality.

Signed-off-by: Konrad Zapałowicz <konrad.zapalowicz@chainguard.dev>
---
 internal/entitlement/client.go | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/internal/entitlement/client.go b/internal/entitlement/client.go
index f53d528..ebb91af 100644
--- a/internal/entitlement/client.go
+++ b/internal/entitlement/client.go
@@ -20,7 +20,7 @@ const (
 	TokenKey = "token"
 
 	// DefaultVendor is the default vendor name.
-	DefaultVendor = "controlplane"
+	DefaultVendor = "chainguard"
 
 	// MarketplaceTypeEnvKey is the environment variable key
 	// that holds the marketplace type.
-- 
2.50.1

